src/tools/PopulateDB.java

```diff
@@ -0,0 +1,34 @@
+package tools;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class PopulateDB {
+    private static void makeTables(Connection conn) {
+        makeUserTable(conn);
```

Comment:
Right now isn't it redundant to have this method as it just passes conn object to makeUserTable? Moreover we should refactor it such that using this method we can call makeUserTable and makeDriverTable as and when required.